lib/api.ts

```diff
@@ -115,13 +130,13 @@ export async function getContributorBySlug(file: string, detail = false) {
   };
 
   const weightedActivity = activityData.activity.reduce(
-    (acc, activity) => {
+    (acc, activity: Activity) => {
```

Comment:
Bro here is type mismatch i don't know why i try to solve it but not able to solve